### PR TITLE
XSH-711 Create github workflow to publish master as alpha version

### PR DESCRIPTION
**Pull request by Dominik-Sauer**

### CODE REVIEW

File: .github/workflows/release-master-as.alpha.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        run: python -m pip install --upgrade pip tbump
+      - name: Bump to alpha
+        run: tbump  --only-patch --non-interactive ${{ github.event.inputs.alpha-release }}
+      - name: Install dependencies
+        run: pip install build
+      - name: Build ${{ matrix.component }}
+        run: |
+          cd ${{ matrix.component }}
+          python -m build
+      - name: Persist ${{ matrix.component }} artifacts
```
Comment (pcerny) [inline .github/workflows/release-master-as.alpha.yaml:39]: Is this step needed? I believe it is needed only if two jobs needs to share data.

**Review by pcerny — CHANGES_REQUESTED** [review]

File: .github/workflows/release-master-as.alpha.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
+name: Release master as alpha
```
Comment (torchbear) [inline .github/workflows/release-master-as.alpha.yaml:1]: Please, mention somewhere in *.md steps how to trigger alpha releases.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Review by pcerny — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-python-sdk-gate-PR128

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-sonarqube-scan-zuul-gate/115/ : SUCCESS in 1m 59s
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-fossa-scanning-zuul-el7/115/ : FAILURE in 11s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-merge-compliance-check/117/ : SUCCESS in 25s

**Pull request merged**